### ISSUE TITLE
The right click seems not to work on some files without extensions

### ISSUE THREAD

**Issue by Nysioko** [files-community/Files#5670]
### Description

Hello,

This morning, trying to open a file with the right click, I noticed a bug. And I don't know if it is referenced, at least I didn't see it so I'm telling you about it.

It's simple, when I right click on a file that has no extension, File only displays a small empty bubble :/

![image](https://user-images.githubusercontent.com/81650517/128007729-96594894-be1a-49ff-a65e-0576d0985bd8.png)
![image](https://user-images.githubusercontent.com/81650517/128007863-395c91ba-a2ea-4daf-af62-ac8b1e906eb3.png)

or when I right-click on a file, and I right-click again on the other one without extension, it opens the menu of the old file where the right-click was done

### Steps To Reproduce

1) Open WSL
2) Go to the desktop for example
3) Write "touch [file name]" in the terminal
4) Open Files and go to the folder where the file was created
5) Right click on it

### Expected behavior

Know how to fix this bug 

### Files Version

1.9.18.0

### Windows Version

Windows 11 Beta Insider - 10.0.22000.100

### Relevant Assets

```markdown
2021-08-03 12:00:41.0097|INFO|OnLaunched|App launched
2021-08-03 12:00:41.6028|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 12:00:41.6697|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 12:00:57.9445|INFO|OnLaunched|App launched
2021-08-03 12:00:58.6197|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 12:00:58.6767|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 12:02:07.3950|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 12:07:09.1438|INFO|OnLaunched|App launched
2021-08-03 12:07:09.8515|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 12:07:09.9332|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 12:07:11.5669|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 12:10:32.6747|INFO|OnLaunched|App launched
2021-08-03 12:10:33.3465|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 12:10:33.3933|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 12:17:13.8839|INFO|OnActivated|App activated
2021-08-03 12:17:14.6954|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 12:17:15.0982|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 12:17:15.4260|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 12:19:12.3599|INFO|OnLaunched|App launched
2021-08-03 12:19:13.0141|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 12:19:13.0990|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 12:19:19.0992|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 12:27:14.6276|INFO|OnLaunched|App launched
2021-08-03 12:27:15.2821|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 12:27:15.3620|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 12:27:16.7559|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 12:29:45.8746|INFO|OnLaunched|App launched
2021-08-03 12:29:46.4787|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 12:29:46.5885|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 12:29:49.1583|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 13:00:47.5910|INFO|OnLaunched|App launched
2021-08-03 13:00:48.3703|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 13:00:48.4243|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 13:00:50.0465|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 13:16:44.3734|INFO|OnLaunched|App launched
2021-08-03 13:16:45.2388|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 13:16:45.3138|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 13:17:47.3172|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 13:24:22.1281|INFO|OnLaunched|App launched
2021-08-03 13:24:22.8082|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 13:24:22.8921|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 13:24:24.4888|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 13:27:37.2586|INFO|OnLaunched|App launched
2021-08-03 13:27:37.9194|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 13:27:37.9945|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 13:27:40.9942|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 13:30:38.6654|INFO|OnLaunched|App launched
2021-08-03 13:30:39.2537|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 13:30:39.3487|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 13:33:56.4821|INFO|OnLaunched|App launched
2021-08-03 13:33:57.1333|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 13:33:57.2091|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 13:33:58.4109|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 13:38:51.6739|INFO|OnLaunched|App launched
2021-08-03 13:38:52.2710|INFO|GetDrivesAsync|Drive added: C:\, Fixed
2021-08-03 13:38:52.3483|INFO|GetDrivesAsync|Drive added: D:\, Fixed
2021-08-03 13:38:53.9360|INFO|IsRunningOnArm|Running on ARM: False
2021-08-03 13:39:14.9045|INFO|OnActivated|App activated
```

**Comment by gave92** [comment]
Thanks for this report. I guess an exception is thrown during context menu items loading.

**Comment by yaichenbaum** [comment]
This issue should be resolved with #5674.
